src/update-repo.ts

```diff
@@ -238,7 +238,7 @@ async function forceCheckout({
 }) {
   try {
     exec(`git checkout ${branch}`, dir)
-    exec(`git reset master --hard`, dir)
+    exec(`git reset main --hard`, dir)
```

Comment:
**question**: Should this line also use the branch variable?

Reply:
This seemed intentional to me. My read of this is that we are looking to ensure we're working with the same code as in `main`, perhaps protecting again any pushes directly the `update-schema` branch? Didn't get a chance to dig too deep into this, so maybe bad logic here?

Reply:
I opened up a PR [here](https://github.com/artsy/update-repo/pull/52) to fix what I think is a bug.